lib/response-builder.ts

```diff
@@ -165,14 +173,14 @@ function collectDockerfilePkgs(
     return;
   }
 
-  return getDockerfileDependencies(dockerAnalysis.dockerfilePackages, deps);
```

Comment:
I'd rather be keen to make any renaming changes in separate "refactor" commits, it makes it a tad cleaner, not a deal breaker in this case though as it's a small change and easy to read, I'll leave it to you!